openlibrary/i18n/__init__.py

```diff
@@ -37,14 +37,20 @@ def _validate_catalog(catalog, locale):
     for message in catalog:
         message_errors = validate(message, catalog)
 
-        if message_errors:
-            if message.lineno:
+        if message.lineno:
+            if message.fuzzy:
+                print(
+                    f'openlibrary/i18n/{locale}/messages.po:'
+                    f'{message.lineno}: {message.string}'
+                )
+            if message_errors:
                 validation_errors.append(
                     f'openlibrary/i18n/{locale}/messages.po:'
                     f'{message.lineno}: {message.string}'
                 )
+            
             for e in message_errors:
-                validation_errors.append(e)
+                validation_errors.append(e)  
```

Comment:
```suggestion
                validation_errors.append(e)
```
Remove trailing whitespace to appease the linter.